DEVDOCS-2204: Fix getImage Handlebars Helpers (#677)
bigcommerce/dev-docs: docs/stencil-docs/reference-docs/handlebars-helpers-reference.md

> ### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)

Reviewer: @aglensmith, linking these headers does not give context into what kind of information the link is leading to. The links should be separated out similar to the Cornerstone example links.

Reviewer 2: I agree with @uwikaiddi. I think we need to change the format of this article and separate the names of the helpers and the associated links. 
```suggestion
### [{{getContentImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImage.js)

```

Author: @uwikaiddi that's not related to this specific fix that nate requested, so it needs to go into a separate ticket.

> 

Reviewer 3: I am curious if the links in the Helper column should also go to github pages like getContentImage. I am wondering why the links are needed in this column.

Author: This isn't related to the fix nate requested, so it should go in a separate ticket and not hold up this pull request.

> Returns URL for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

Reviewer 2: ```suggestion
Returns URL for an image uploaded to the WebDAV **content** folder. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

#### Parameters

* `a` **{String}:** Image path relative to `/dav/content/`

> * `width` **{Int}:** Width in pixels

Reviewer 4: Might be worth mentioning that both width and height are optional.

Validation combinations are:
- No height/No width (returns original image)
- Width only
- Width + height

Not valid:
- Height only

This is true for both non-srcset helpers.

Reviewer 4: Never mind, I see this is covered a bit later on.

* `height` **{Int}:** Height in pixels

#### Example

```handlebars
<!-- Original image URL returned if no size or invalid size is passed in -->
{{getContentImage "asset.jpg"}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/original/content/asset.jpg  -->

{{getContentImage "asset.jpg" width="a" height="a"}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/original/content/asset.jpg  -->

<!-- Sized image URL returned if valid height and/or width passed in -->
{{getContentImage "asset.jpg" width=123 height=321}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123x321/content/asset.jpg -->

{{getContentImage "asset.jpg" width=123}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123w/content/folder/asset.jpg -->
```

### [{{getContentImageSrcset}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getContentImageSrcset.js)

> Returns a `srcset` for an image [uploaded to `/dav/content/`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

Reviewer 2: ```suggestion
Returns a `srcset` for an image uploaded to `/dav/content/`. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

{{getImage image "thumbnail"}}
```

[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getImage).

### [{{getImageManagerImage}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getImageManagerImage.js)

> Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image URL for an image [uploaded to `/dav/product_images/uploaded_images`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

Reviewer 2: ```suggestion
Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image URL for an image uploaded to `/dav/product_images/uploaded_images`. For more information on using WebDAV, see [File Access (WebDAV)](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

```

* `a` **{String}:** Image path relative to `/dav/product_images/uploaded_images`
* `width` **{Int}:** Width in pixels
* `height` **{Int}:** Height in pixels

#### Example

```handlebars
<!-- Original image URL returned if no size or invalid size is passed in -->
{{getImageManagerImage "asset.jpg"}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/original/image-manager/asset.jpg  -->

{{getImageManagerImage "folder/asset.jpg" height=123}} <!-- height must be accompanied by width -->
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/original/image-manager/folder/asset.jpg  -->

<!-- Sized image URL returned if valid height and/or width passed in -->
{{getImageManagerImage "asset.jpg" width=123}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123w/image-manager/asset.jpg -->

{{getImageManagerImage "folder/asset.jpg" width=123 height=321}}
<!--=> https://cdn.bcapp/3dsf74g/images/stencil/123x321/image-manager/folder/asset.jpg -->
```

[See it in Cornerstone](https://github.com/bigcommerce/cornerstone/search?l=HTML&q=getImageManagerImage).

### [{{getImageManagerImageSrcset}}](https://github.com/bigcommerce/paper-handlebars/blob/master/helpers/getImageManagerImageSrcset.js)

> Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image `srcset` for an image [uploaded to `/dav/product_images/uploaded_images`](https://support.bigcommerce.com/s/article/File-Access-WebDAV).

Reviewer 2: ```suggestion
Returns an [Image Manager](https://support.bigcommerce.com/s/article/Using-the-Image-Manager) image `srcset` for an image uploaded to `/dav/product_images/uploaded_images`.

```
